<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>219</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>1)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dASpin"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>x&lt;sup&gt;2&lt;/sup&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dBSpin"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>x +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dCSpin"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>= 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enterBtn">
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dASpin_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>x&lt;sup&gt;2&lt;/sup&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dBSpin_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>x +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dCSpin_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>= 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enterBtn_2">
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
        </item>
        <item>
         <widget class="QPushButton" name="multPolyBtn">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="inversePolyBtn">
          <property name="text">
           <string>!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="X1">
          <property name="text">
           <string>x1 =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="X2">
          <property name="text">
           <string>x2 = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calcRootsBtn">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calcIntegralBtn">
          <property name="text">
           <string>Calc. Integral</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="integr">
          <property name="text">
           <string>Integral</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="curriedFuncBtn">
          <property name="text">
           <string>Curried func</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curriedFunc">
          <property name="text">
           <string>Curried func</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>x +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Sum:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="polySum_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ax &lt;span style=&quot; vertical-align:super;&quot;&gt;2  &lt;/span&gt;+ bx + c = 0 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Subtraction:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="polyMinus_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ax &lt;span style=&quot; vertical-align:super;&quot;&gt;2  &lt;/span&gt;+ bx + c = 0 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="equality_label">
          <property name="text">
           <string>Equality</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calcOtherBtn">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="PolyA_label">
        <property name="minimumSize">
         <size>
          <width>99</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ax &lt;span style=&quot; vertical-align:super;&quot;&gt;2  &lt;/span&gt;+ bx + c = 0 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="showPolyA">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Show poly A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PolyB_label">
        <property name="minimumSize">
         <size>
          <width>99</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ax &lt;span style=&quot; vertical-align:super;&quot;&gt;2  &lt;/span&gt;+ bx + c = 0 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="showPolyB">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Show poly B</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="PolyA_radio">
        <property name="text">
         <string>Poly A</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="PolyB_radio">
        <property name="text">
         <string>Poly B</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Choose polynom to work with</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
